CONTRIBUTING.md

```diff
@@ -34,9 +32,12 @@ There are three types of issue templates:
 * Run `npm run lint` to conform to our lint rules
 
 ## License
-By contributing to Draft.js, you agree that your contributions will be licensed
-under MIT license.
+By contributing to EddieBotUI, you agree that your contributions will be licensed
+under [MIT license](./LICENSE).
+
+## Need help?
+For **help**, **support** or **questions** please use our [community chat](https://discord.com/invite/jZQs6Wu) on Discord.
 
 ---
 
-For **help**, **support** or **questions** please use our [community chat](https://discord.com/invite/jZQs6Wu) 🚑.
+Special thanks go to the [Draft.js team](https://github.com/facebook/draft-js) which inspired us with this template.
```

Comment:
Is this meant to be `Draft.js`?

Reply:
Yes @eddiejaoude, I used their template. Clean and with essential things. We can simply add things to it :)

Reply:
Ok 👍